### DISCUSSION TITLE
HC_15_param?

### DISCUSSION THREAD

**Discussion by Suedschwede66** [Arnold-n/P1P2MQTT#155] [Q&A]
Whats this for Parmeterlist?

![Bildschirmfoto vom 2024-12-30 17-26-48](https://github.com/user-attachments/assets/0651b7e6-7686-4725-82f0-cb1d1a90d9ad)

**Comment by Arnold-n** [comment]
We're trying to find out. A few are known and will be renamed in a future version. Some are not even temperatures, and some are duplicates so not useful (like nr 6 which is defrost_activity).

**Reply by Suedschwede66** [reply]
I think :)

Param15_Temperature_0A = R2T-Verdichtungsendtemperatur (Ausseneinheit)
sensor.hc_unknown_temperature_unused_15_0 = R9T-Wassereintrittstemp. vor PlattenWaermeTauscher (Ausseneinheit)
sensor.hc_unknown_temperature_unknown_q = R10T-Wasseraustrittstemp. nach PlattenWaermeTauscher (Ausseneinheit)
